examples/demo/src/shared/utils/_vars.scss

```diff
@@ -1,5 +1 @@
-@import '../../../node_modules/@axa-fr/react-toolkit-core/dist/assets/scss/_custom-af.scss';
-@import '../../../node_modules/@axa-fr/react-toolkit-core/dist/assets/scss/_custom-bootstrap';
-@import '../../../node_modules/bootstrap/scss/_functions.scss';
-@import '../../../node_modules/bootstrap/scss/_mixins.scss';
-@import '../../../node_modules/bootstrap/scss/_variables.scss';
+@import '../../../node_modules/@axa-fr/react-toolkit-core/dist/assets/scss/core';
```

Comment:
Great job Samuel !

is it possible to set up non relative links?
like "@axa-fr/react-toolkit-core" instead of '../../../node_modules/@axa-fr/react-toolkit-core/'

Reply:
Thank you @guillaumechervetaxa  :)
This file is no longer used, i deleted this.
I also had overridable variables scss in my last commit